src/tests/test.spec.js

```diff
@@ -0,0 +1,16 @@
+import { test} from '@playwright/test';
+import { LoginPage } from "../pages/Login.page";
+import { InventoryPage } from "../pages/Inventory.page";
+import { BasePage } from '../pages/Base.page';
+
+test('Sorting',
```

Comment:
Lets check different sorting options, not only z-a. And test name should be more specific, like "Check sorting products on the products page: z-a" for example.